packages/fxa-settings/src/components/Nav/index.test.tsx

```diff
@@ -56,15 +56,34 @@ describe('Nav', () => {
     );
 
     expect(screen.queryByTestId('nav-link-subscriptions')).toBeNull();
+
+    // TODO: shouldn't be null in FXA-4106, replace w/ DataCollection test
+    expect(screen.queryByTestId('nav-link-data-collection')).toBeNull();
+  });
+
+  it('conditionally renders DataCollection', () => {
+    render(
+      <AppContext.Provider value={{ account, config: getDefault() }}>
+        <Nav showDataCollection />
+      </AppContext.Provider>
+    );
+
+    expect(screen.getByTestId('nav-link-data-collection')).toHaveTextContent(
```

Comment:
I added some explicit checks to be sure users wouldn't see any of this stuff until FXA-4106 and scattered those TODO comments about to make them easy to find all relevant files for when working on that ticket. Though, I should probably add one to the "with telemetry opt-out" story, will do that with any other edits.